### ISSUE TITLE
NullPointerException raised on multi search

### ISSUE THREAD

**Issue by nkconnor** [sksamuel/elastic4s#1494]
I think this is related to #1171. 

```
       // SearchImplicits.scala:43
        val json = JacksonSupport.mapper.readTree(response.entity.get.content)
        val items = json
          .get("responses")
```
the response handler tries to `get` the non-existent "responses", so:

```
java.lang.NullPointerException
        at com.sksamuel.elastic4s.http.search.SearchImplicits$MultiSearchHttpExecutable$$anon$2.handle(SearchImplicits.scala:43)
        at com.sksamuel.elastic4s.http.search.SearchImplicits$MultiSearchHttpExecutable$$anon$2.handle(SearchImplicits.scala:39)
        at com.sksamuel.elastic4s.http.HttpClient.$anonfun$execute$1(HttpClient.scala:69)
        at scala.util.Success.$anonfun$map$1(Try.scala:251)
        at scala.util.Success.map(Try.scala:209)
        at scala.concurrent.Future.$anonfun$map$1(Future.scala:288)
        at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:29)
        at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:29)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:60)
        at akka.dispatch.BatchingExecutor$AbstractBatch.processBatch(BatchingExecutor.scala:55)
        at akka.dispatch.BatchingExecutor$BlockableBatch.$anonfun$run$1(BatchingExecutor.scala:91)
        at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
        at scala.concurrent.BlockContext$.withBlockContext(BlockContext.scala:81)
        at akka.dispatch.BatchingExecutor$BlockableBatch.run(BatchingExecutor.scala:91)
        at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:40)
        at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(ForkJoinExecutorConfigurator.scala:43)
        at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
        at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
        at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

I can't get Multi Search to work in general.. will try to work on getting another issue together for it

**Comment by sksamuel** [comment]
Can you show me the query / response that caused the NPE ?

**Comment by sksamuel** [comment]
Fixed in 7.0
